src/config/statsd.ts

```diff
@@ -11,14 +11,19 @@ export const globalTags = {
   region: process.env.MICROS_AWS_REGION || 'localhost',
 };
 
-const logger = getLogger('statsd');
+const logger = getLogger('config.statsd');
 
 const statsd = new StatsD({
   prefix: 'github-for-jira.',
   host: 'platform-statsd',
   port: 8125,
   globalTags,
-  errorHandler: (err) => logger.warn({ err }, 'error writing metrics'),
+  errorHandler: (err) => {
+    if (process.env.NODE_ENV !== 'development') {
+      logger.warn(err, 'error writing metrics')
```

Comment:
I've added a check to not log these errors when we test locally, hence there will be a lot